src/Mod/Sketcher/Gui/TaskSketcherGeneral.cpp

```diff
@@ -45,12 +45,6 @@ SketcherGeneralWidget::SketcherGeneralWidget(QWidget *parent)
     ui->renderingOrder->setSizeAdjustPolicy(QAbstractScrollArea::AdjustToContents);
 
     // connecting the needed signals

```

Comment:
As I said, I'm not in favor of deleting all from TaskSketcherGeneral.
I think both should work aside each other. This will:
* Allow to check things are well coded :)
* Offer possibility to user to use grid in the most appropriate way (later when all options of TaskSketcherGeneral are transfered, we can hide it by default and propose it as convenience panel)